test/support/redis_mock.ex

```diff
@@ -20,6 +20,14 @@ defmodule OffBroadwayRedisStream.RedisMock do
     :ok
   end
 
+  @impl true
+  def delete_message(ids, config) do
+    info = %{ids: ids, pid: self()}
+    Super.ack(ids, config)
```

Comment:
Should be `Super.delete_message`.
Let's also add a test for deletion?

Reply:
I've added a test for the `delete_on_acknowledgment` flag which encompasses this I think